### PR TITLE
reorder the unpublished judgments metadata

### PR DESCRIPTION
**Pull request by Donna-H**
## Changes in this PR:
Reorder the unpublished judgments metadata
## Trello card / Rollbar error (etc)
https://trello.com/c/gOaBkLDl/1249-eui-to-re-order-the-unpublished-judgments-metadata
### Before
before-order

### After
after-order

- [ ] Requires env variable(s) to be updated

### CODE REVIEW

**Comment by Donna-H** [comment]
[[EUI] To re-order the unpublished judgments metadata](https://trello.com/c/gOaBkLDl/1249-eui-to-re-order-the-unpublished-judgments-metadata)

File: package.json [hunk 1]
```diff
@@ -26,7 +26,7 @@
     "@babel/core": "^7.17.2",
     "@babel/preset-env": "^7.16.11",
     "babel-loader": "^9.0.0",
-    "cypress": "^12.17.4",
+    "cypress": "^4.2.0",
```
Comment (anthonyhashemi) [inline package.json:29]: was this from the npm audit command you ran?

I think we should revert this and the package.json change as they arent needed for this PR

**Review by anthonyhashemi — CHANGES_REQUESTED** [review]
Can you undo the package.json and package-lock.json since they arent needed for this change?

The actual change looks good though 👍

**Review by Terry-Price — CHANGES_REQUESTED** [review]
Hi @Donna-H please can you get rid of the two colons used in the labels as discussed.
Otherwise all good for now.

**Comment by Donna-H** [comment]
> Hi @Donna-H please can you get rid of the two colons used in the labels as discussed. Otherwise all good for now.

@Terry-Price okay removed and updated the before and after images

**Review by anthonyhashemi — APPROVED** [review]

**Pull request merged**